plugins/destination/clickhouse/client/spec.go

```diff
@@ -47,6 +52,21 @@ func (s *Spec) SetDefaults() {
 	if s.Engine == nil {
 		s.Engine = queries.DefaultEngine()
 	}
+
+	if s.BatchSize == 0 {
+		const defaultBatchSize = 10_000 // 10K
+		s.BatchSize = defaultBatchSize
+	}
+
+	if s.BatchSizeBytes == 0 {
+		const defaultBatchSizeBytes = 50 << 20 // 50 MiB
```

Comment:
Should it be 4MB ? otherwise memory can blow up as it's per table

Reply:
I can make it 5 MiB (or 4, if you insist)

Reply:
5c0e1150d394f877575bb0920d46118afcbf13b6